### PR TITLE
feat(tokens): update visited tokens

### PR DESCRIPTION
**Pull request by booc0mtaco**
- add text-utility-interactive-primary-visited
- add text-utility-interactive-secondary-visited
- add icon-utility-interactive-primary-visited
- add icon-utility-interactive-secondary-visited
- mark text-utility-interactive-visited as deprecated
- mark icon-utility-interactive-visited as deprecated
- change text-utility-inverse-interactive-visited from purple-350 to white
- update usages of old tokens to use the new tokens

### Test Plan:

- [ ] Wrote/updated [automated tests](https://czi.atlassian.net/wiki/x/Hbl1H)
- [ ] CI tests / new tests are not applicable
- [ ] Manually tested my changes, and here are the details:

### CODE REVIEW

File: src/components/Link/Link.module.css [hunk 1]
```diff
@@ -114,7 +114,7 @@
 }
 
 .link:visited {
-  color: var(--eds-theme-color-text-utility-interactive-visited);
+  color: var(--eds-theme-color-text-utility-interactive-primary-visited);
```
Comment (booc0mtaco) [inline src/components/Link/Link.module.css:117]: > update usages of old tokens to use the new tokens

File: src/design-tokens/themes.json [hunk 2]
```diff
@@ -1699,6 +1699,10 @@
                 "active": {
                   "value": "{eds.theme.color.text.utility.interactive.primary.active}",
                   "group": "color"
+                },
+                "visited": {
+                  "value": "{eds.theme.color.text.utility.interactive.primary.visited}",
+                  "group": "color"
```
Comment (booc0mtaco) [inline src/design-tokens/themes.json:1705]: > add icon-utility-interactive-primary-visited

File: src/design-tokens/themes.json [hunk 3]
```diff
@@ -1713,11 +1717,16 @@
                 "active": {
                   "value": "{eds.theme.color.text.utility.interactive.secondary.active}",
                   "group": "color"
+                },
+                "visited": {
+                  "value": "{eds.theme.color.text.utility.interactive.secondary.visited}",
+                  "group": "color"
```
Comment (booc0mtaco) [inline src/design-tokens/themes.json:1723]: > add icon-utility-interactive-secondary-visited

File: src/design-tokens/themes.json [hunk 4]
```diff
@@ -1713,11 +1717,16 @@
...
                 }
               },
               "visited": {
                 "value": "{eds.theme.color.text.utility.interactive.visited}",
-                "group": "color"
+                "group": "color",
+                "comment": "@deprecated This should not be used in code or design. It will be removed in a future version of EDS."
```
Comment (booc0mtaco) [inline src/design-tokens/themes.json:1729]: > mark icon-utility-interactive-visited as deprecated

File: src/design-tokens/themes.json [hunk 5]
```diff
@@ -2033,6 +2042,9 @@
                 },
                 "active": {
                   "value": "{eds.color.blue.950}"
+                },
+                "visited": {
+                  "value": "{eds.color.blue.850}"
```
Comment (booc0mtaco) [inline src/design-tokens/themes.json:2047]: > add text-utility-interactive-primary-visited

File: src/design-tokens/themes.json [hunk 6]
```diff
@@ -2044,10 +2056,14 @@
                 },
                 "active": {
                   "value": "{eds.color.blue.750}"
+                },
+                "visited": {
+                  "value": "{eds.color.blue.550}"
```
Comment (booc0mtaco) [inline src/design-tokens/themes.json:2061]: > add text-utility-interactive-secondary-visited

File: src/design-tokens/themes.json [hunk 7]
```diff
@@ -2044,10 +2056,14 @@
...
                 }
               },
               "visited": {
-                "value": "{eds.color.purple.550}"
+                "value": "{eds.color.purple.550}",
+                "comment": "@deprecated This token is deprecated. Please use eds-theme-color-border-brand-primary-default instead"
```
Comment (booc0mtaco) [inline src/design-tokens/themes.json:2066]: > mark text-utility-interactive-visited as deprecated

File: src/design-tokens/themes.json [hunk 8]
```diff
@@ -2111,7 +2127,7 @@
               },
               "interactive": {
                 "visited": {
-                  "value": "{eds.color.purple.350}"
+                  "value": "rgb(var(--eds-color-white) / 1)"
```
Comment (booc0mtaco) [inline src/design-tokens/themes.json:2130]: > change text-utility-inverse-interactive-visited from purple-350 to white

**Review by ahuth — APPROVED** [review]

**Pull request merged**